README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTelemetry Java Contrib
+
+This project is intended to provide helpful libraries and standalone OpenTelemetry-based utilities that don't fit
+the express scope of the [OpenTelemetry Java](https://github.com/open-telemetry/opentelemetry-java) or
+[Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation) projects.  If you need an
+easier way to bring observability to remote JVM-based applications and workflows that isn't easily satisfied by an SDK
+feature or via bytecode instrumentation, this project is hopefully for you.
```

Comment:
opentelemetry-java-instrumentation is not only for bytecode instrumentation
```suggestion
feature or via instrumentation, this project is hopefully for you.
```

Reply:
of course, corrected.